### DISCUSSION TITLE
Garbled text missing spaces after opening ebook on Calibre

### DISCUSSION THREAD

**Discussion by milady495** [noDRM/DeDRM_tools#305] [Q&A] [answered]
Hi! I recently bought an ebook from Amazon (I know, bad idea, but I couldn't find it anywhere else) and after adding it to the Calibre library I opened the viewer and the text appeared all jumbled up, without spaces, some of it in a smaller font... Just a very messy format, very uncomfortable to read.

Some images to show the issue:
![image](https://user-images.githubusercontent.com/128744647/227331577-6f510c84-9926-4508-8fad-5ca1bbb7fe48.png)
![image](https://user-images.githubusercontent.com/128744647/227331979-ae7b1c31-d3ac-45d1-b68d-b4b38ef38378.png)

It also appears like it has some badly scanned images:
![image](https://user-images.githubusercontent.com/128744647/227331929-a8d009a5-4ca3-4fb6-a299-a2cb176df8b6.png)

It was a book published March 2023, so I couldn't download it from Kindle for PC 1.17. I got it via the download option for transfering through USB (first time I ever do that).

My Calibre version is 5.38.0, DeDRM 10.0.3.

Did I do something wrong or is this some sort of issue that is being worked on?

Thank you for your amazing work with this plugin :)

**Comment by stillhope** [comment, marked as answer]
Hi: You have done nothing wrong. 

This looks to be an example of one of the Amazon ebook scams - fake book.  It started about 1.5 years ago and for often out of print, or hard to get titles. I must of reported over 100. Amazon is doing nothing to stop it.  The scammer takes a scan book (typically available for free from internet archive) converts to EPUB, wraps it in kindle and publishes it on amazon.

So far for everyone I have reported, the meta data was incorrect e.g. not from original publisher.  The only ones that have been removed so far of the 100 are those where I took the time to leave a bad quality review, and others who also got caught, did the same.

Suggest leave a review saying terrible copy and needs to be removed. Request a refund if you like.

**Comment by ElleKayEm** [comment]
Not a DRM issue.  It wouldn't even open with calibre's ebook viewer if it had DRM.

**Comment by milady495** [comment]
Oh, thank you both for your answer, I was going crazy searching online and trying different things with my Calibre... 

I read most books these days directly from Calibre so I didn't even think of trying it on the Kindle and seeing if the ebook itself was the problem. Hope this helps other people that come across this problem.
